training/src/main/scala/com/airbnb/aerosolve/training/AdditiveModelTrainer.scala

```diff
@@ -299,17 +311,6 @@ object AdditiveModelTrainer {
     return loss
   }
 
-  private def transformExamples(input: RDD[Example],
-                        config: Config,
-                        key: String,
-                        params: AdditiveTrainerParams): RDD[Example] = {
-    if (params.isRanking) {
-      LinearRankerUtils.transformExamples(input, config, key)
```

Comment:
when `makePointwiseFloat` is used one Example contains only one example, but when `transformExamples` is used one Example can contain multiple examples and it is used for compute ranking loss, if you look at the `getTrainingExamples` in PricingPipeline, you can see how examples are grouped. As for context, I think for both cases we are not using it.